OctoKit/Issue.swift

```diff
@@ -9,6 +9,55 @@ public enum Openness: String, Codable {
     case All = "all"
 }
 
+#if os(Linux)
+open class Issue: Codable {
```

Comment:
is it possible that the if statements can just wrap the first line of the definition?

Reply:
It needs to remove also the `@objc` variables,
I tried anyway to not put all the class inside the if but just the top part that has the first line plus the `objc` variables, but this is what happens

<img width="1043" alt="screenshot 2018-11-06 at 19 18 25" src="https://user-images.githubusercontent.com/17830956/48088074-ec748b80-e1f8-11e8-95df-4a15e1c761ee.png">

Reply:
Thanks!